x-pack/plugins/ml/public/application/model_management/test_models/models/inference_input_form/input_form_controls.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+
+import { EuiButton, EuiButtonEmpty, EuiFlexItem } from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useTestTrainedModelsContext } from '../../test_trained_models_context';
+import type { InferrerType } from '..';
+
+interface Props {
+  testButtonDisabled: boolean;
+  createPipelineButtonDisabled: boolean;
+  inferrer: InferrerType;
+}
+
+export const InputFormControls: FC<Props> = ({
+  testButtonDisabled,
+  createPipelineButtonDisabled,
+  inferrer,
+}) => {
+  const testTrainedModelsContext = useTestTrainedModelsContext();
+
+  return (
+    <>
+      <EuiFlexItem grow={false}>
+        <EuiButton
+          disabled={testButtonDisabled}
+          fullWidth={false}
+          data-test-subj={'mlTestModelTestButton'}
+          type={'submit'}
+        >
+          <FormattedMessage
+            id="xpack.ml.trainedModels.testModelsFlyout.inferenceInputForm.runButton"
+            defaultMessage="Test"
+          />
+        </EuiButton>
+      </EuiFlexItem>
+      {!testTrainedModelsContext?.currentContext.createPipelineFlyoutOpen ? (
+        <EuiFlexItem grow={false}>
```

Comment:
`testTrainedModelsContext` is always defined, so the `?` is not needed`

Reply:
Updated the way we access context in c582e99a7ece7292c11667215cdffd931c2b8a6a